### PR TITLE
Import %>% and .data to avoid NOTE about undefined global variables

### LINKED ISSUE #36: missing imports
See example:
```
synapseforms::get_submissions_metadata(syn, form_group$groupId)
Error in data_df %>% dplyr::mutate(submissionStatus_submittedOn = submissionStatus$submittedOn,  : 
  could not find function "%>%"
```

### PR DESCRIPTION
**Pull request by karawoo**
Fixes #36

### CODE REVIEW

**Comment by Aryllen** [comment]
Thanks for fixing the pipe import. I'm going to check this out a little later today since I have a meeting coming up, but I tried using `.data` when I originally made this function and, at the time, it made the function not work correctly.

**Comment by karawoo** [comment]
Hmm ok, I tried it out and it looked ok to me. If it doesn't work for you, could we add a test for whatever behavior is broken by using `.data`?

**Comment by philerooski** [comment]
I don't think it's necessary to import `%>%` at the function level.

Putting `importFrom(magrittr,"%>%")` in the NAMESPACE and `magrittr` in the `imports` section of `DESCRIPTION` should be enough iirc.

**Comment by karawoo** [comment]
I'm not sure I follow your comment @philerooski, I think that's what I've done. My NAMESPACE is auto-generated from roxygen2 comments, so `#' @importFrom magrittr %>%` needs to appear somewhere in those comments.

**Comment by karawoo** [comment]
Not sure what the failing tests here are about, I don't see any failures locally. Will investigate...

**Comment by karawoo** [comment]
I think I've fixed the source of the failing tests, which was caused by the latest version of tibble being stricter about types within columns of a tibble.

File: R/get-submission-metadata.R [hunk 1]
```diff
@@ -67,6 +67,8 @@ get_submissions_metadata <- function(syn, group, all_users = TRUE,
 #'
 #' @keywords internal
 #' @param data The json submission metadata.
+#' @importFrom magrittr %>%
```
Comment (philerooski) [inline R/get-submission-metadata.R:70]: Function-level lines like this was what I was referring to. Are these auto-generated by roxygen? If not, I think `importFrom(magrittr,"%>%")` in the NAMESPACE and specifying magrittr in the imports is sufficient.

Comment (karawoo) [reply R/get-submission-metadata.R:70]: The comments are not auto generated by roxygen, but they are what roxygen uses to generate the NAMESPACE. So they need to be here if we want it added to the NAMESPACE.

Comment (philerooski) [reply R/get-submission-metadata.R:70]: Ah, interesting. So just a single inclusion (in a single function's docstring) is enough for Roxygen to put pipe in the package namespace, and thus make it available to all functions? Quirky 🤷

Comment (karawoo) [reply R/get-submission-metadata.R:70]: yup

**Review by Aryllen — APPROVED** [review]
This is definitely working. I feel like I was using it the same way when I tried previously, but there must have been something I was doing wrong. I vaguely remember wanting to ask you about it, but it apparently slipped my mind.

Approving and merging. The test that's failing is for Suggests packages, which are really only in there for my own convenience. I'll open a new branch and remove those from the DESCRIPTION file.

**Pull request merged**